src/test/librados/c_read_operations.cc

```diff
@@ -470,7 +470,7 @@ TEST_F(CReadOpsTest, Exec) {
   uint64_t features;
   EXPECT_EQ(sizeof(features), bytes_read);
   // make sure buffer is at least as long as it claims
-  ASSERT_TRUE(out[bytes_read-1] == out[bytes_read-1]);
+  ASSERT_TRUE(out[0] = out[bytes_read-1]);
```

Comment:
no, this is wrong. we cannot be sure that `out[0]` evaluates as true.

Reply:
all test failures of http://pulpito.ceph.com/kchai-2018-01-25_14:55:14-rados-wip-kefu-testing-2018-01-24-1041-distro-basic-mira/ are related to this change.

Reply:
Oops, my bad, sorry for wasting your time! Let me try something else.